meilisearch-types/Cargo.toml

```diff
@@ -9,9 +9,10 @@ actix-web = { version = "4.0.1", default-features = false }
 csv = "1.1.6"
 either = { version = "1.6.1", features = ["serde"] }
 enum-iterator = "0.7.0"
-milli = { git = "https://github.com/meilisearch/milli.git", tag = "v0.33.0" }
+milli = { git = "https://github.com/meilisearch/milli.git", branch = "update-dependencies" }
 proptest = { version = "1.0.0", optional = true }
 proptest-derive = { version = "0.3.0", optional = true }
+roaring = { version = "0.10.0", features = ["serde"] }
```

Comment:
noice

Reply:
Because now we store a bitmap in the `Details` or `KindWithContent` I already forgot, and we need to write this in heed with the `SerdeJson` codec.